docs/quick_start.md

```diff
@@ -163,11 +141,11 @@ The metadata provides information about the project such as the name, author and
 }
```

Comment:
I guess the dependencies here are just an example and not actually the dependencies since they update... I guess this is okay